airflow/contrib/kubernetes/secret.py

```diff
@@ -39,3 +39,10 @@ def __init__(self, deploy_type, deploy_target, secret, key):
             self.deploy_target = deploy_target
         self.secret = secret
         self.key = key
+
+
+class SecretObject:
```

Comment:
It's not clear to me why this class is useful if it only contains a single `name` member.

Reply:
@pgagnon removed as for now, to simplify this.